eip-4363.md

```diff
@@ -0,0 +1,30 @@
+---
+eip: 4363
+title: Transaction index OpCode
+author: Edgar (@fxfactorial), Dean (@decanus)
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-10-12
+---
```

Comment:
It appears you're missing the `description` and `discussions-to` fields of the preamble.